app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.systers.mentorship.view.activities
+
+import android.content.Intent
+import android.content.res.Resources
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_about.*
+import org.systers.mentorship.R
+
+class AboutActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_about)
+
+        btnGit.setOnClickListener {
+            val string = R.string.url_github
+            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(string.toString())))
```

Comment:
you can put this string right in this line, as you did with the following links, but calling `getString()` method, like:
```
getString(R.string.url_github)
```